controllers/resources/templates.go

```diff
@@ -63,6 +65,15 @@ func (m *manager) GetCurrentTemplateWithTimeout(node *v1.Node, nhc *remediationv
 	return nil, nil, NoTemplateLeftError{msg: fmt.Sprintf("didn't find a template to use for NHC %s and node %s", nhc.Name, node.Name)}
 }
 
+func (m *manager) GetTemplate(mhc *machinev1beta1.MachineHealthCheck) (*unstructured.Unstructured, error) {
```

Comment:
Fair enough, I thought it could help a little for readability when now a remediation template could be associated with an NHC or MHC, but it is only that important.